### PR TITLE
Update to clean up the dependency trees

### PR DESCRIPTION
**Pull request by prb112**
- Remove unused surface area. 

- Address 
fhir-persistence-blob:
kotlin-stdlib-1.3.72.jar (pkg:maven/org.jetbrains.kotlin/kotlin-stdlib@1.3.72, cpe:2.3:a:jetbrains:kotlin:1.3.72:*:*:*:*:*:*:*) : CVE-2020-29582
kotlin-stdlib-common-1.3.70.jar (pkg:maven/org.jetbrains.kotlin/kotlin-stdlib-common@1.3.70, cpe:2.3:a:jetbrains:kotlin:1.3.70:*:*:*:*:*:*:*) : CVE-2020-29582
- also an issue in bulkdata.

Signed-off-by: Paul Bastide <pbastide@us.ibm.com>

### CODE REVIEW

File: fhir-parent/pom.xml [hunk 1]
```diff
@@ -641,6 +641,16 @@
...
+            <dependency>
+                <groupId>org.jetbrains.kotlin</groupId>
+                <artifactId>kotlin-stdlib</artifactId>
+                <version>1.6.10</version>
+            </dependency>
+            <dependency>
+                <groupId>org.jetbrains.kotlin</groupId>
+                <artifactId>kotlin-stdlib-common</artifactId>
+                <version>1.6.10</version>
+            </dependency>
```
Comment (lmsurpre) [inline fhir-parent/pom.xml:653]: maybe a comment indicating these are used in azure blob sdk from fhir-persistence-blob?

File: fhir-parent/pom.xml [hunk 2]
```diff
@@ -641,6 +641,16 @@
                 <artifactId>azure-core</artifactId>
                 <version>1.24.0</version>
             </dependency>
+            <dependency>
```
Comment (prb112) [inline fhir-parent/pom.xml:644]:
```suggestion
            <!-- Used in OKHTTP as part of azure-core -->
            <dependency>
```

**Review by lmsurpre — APPROVED** [review]
Assuming it still works after the exclusions, then LGTM.

**Comment by prb112** [comment]
> Assuming it still works after the exclusions, then LGTM.

It should since it's part of the original dep.

**Pull request merged**